CommunityToolkit.Graph.Uwp/Controls/PersonView/PersonView.xaml

```diff
@@ -64,30 +64,76 @@
                             </Grid>
                         </Grid>
 
-                        <controls:SwitchPresenter
-                            Grid.Column="1"
-                            Value="{Binding PersonViewType, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource ObjectToStringConverter}}"
-                            VerticalAlignment="Center">
-                            <!-- One line -->
-                            <controls:Case Value="OneLine">
-                                <TextBlock Margin="6,-2,2,0"
-                                    FontWeight="SemiBold"
-                                    Text="{Binding PersonDetails.DisplayName, RelativeSource={RelativeSource TemplatedParent}}" />
-                            </controls:Case>
+                        <Grid Grid.Column="1">
```

Comment:
Can't we just set the Column on both the children rather than introducing another Grid layer?

Reply:
Definitely. I found another bug in that file as well. I'll fix them at the same time :)